cmd/svcat/plugin/manifest.go

```diff
@@ -70,6 +70,9 @@ type Plugin struct {
 	// Name of the command for the help text. Required.
 	Name string `yaml:"name"`
 
+	// Use is the one-line description of how the command is used.
+	Use string `yaml:"use"`
```

Comment:
This is not a supported field in the kubectl plugin format and should not be added.

https://github.com/kubernetes/kubernetes/blob/e85b81bbee098a7ec75cc894a8785867ec586798/pkg/kubectl/plugins/plugins.go#L47-L55

Reply:
Ah thanks, that's why I needed a link to the PR. I didn't realize that Use was added in it. 👍